### DISCUSSION TITLE
How can I use Sign in with Apple for web in debug mode on local machine?

### DISCUSSION THREAD

**Discussion by AlexSeednov** [aboutyou/dart_packages#428] [Q&A] [answered]
Hey! 

Apple requires to add web domain with working HTTPS in Service ID, so I didn't fine how to fully test Sign in with Apple flow for web locally - it doesn't redirect in the end of the authorization flow.

So is it possible? Sign in with Google provide possibility to add localhost in approved domains, for example, but Apple doesn't, as I understood.

Thank you for your help and time, every help will be useful.

**Comment by tp** [comment, marked as answer]
You can not test with `localhost`. But instead you can just pick any domain and map it to "localhost" on your machine.

This is how we test the plug-in during development:

https://github.com/aboutyou/dart_packages/blob/master/packages/sign_in_with_apple/sign_in_with_apple_web/Development.md

**Reply by AlexSeednov** [reply]
Awesome, thank you a lot!
